resources/main.css

```diff
@@ -48,6 +48,9 @@
 	color: red;
 	font-weight: bold;
 }
-.mw-scratch-confirmaccount-click-copy {
+#mw-scratch-confirmaccount-click-copy {
 	font-weight: normal;
 }
+.jsonly noscript style{
```

Comment:
Is this even possible?

Reply:
It is (because we're tagging it with priority as far as I know).

Reply:
From what I can tell this isn't. You will need to inject this CSS into the head using the appropriate MediaWiki libraries.

Reply:
I should say what the semantics of this rule do:
They apply the rule to a `<style>` tag inside a `<noscript>` tag inside a tag with a class of jsonly. You shouldn't apply CSS rules directly to a `<style>` at all tag. Also, please test code before submitting a PR.